modules/ROOT/pages/dw-runtime-functions-try.adoc

```diff
@@ -68,5 +68,8 @@ When `randomNumber` succeeds, the output is:
   "result": "OK"
 }
 ----
-Based on the output of the `try` function, you can add conditional logic to execute when the result is `success: true` or `success: false`.
+
+The `orElseTry` and `orElse` functions will also continue processing if the `try` function fails. See the `orElseTry` and `orElse` documentation for more complete examples of handling failing `try` function expressions.  
+
+Note: Instead of using the `orElseTry` and `orElse` functions, based on the output of the `try` function, you can add conditional logic to execute when the result is `success: true` or `success: false`, but typically this will be unneccessary. 
```

Comment:
I'm not sure about this `but typically this will be unneccessary`, do we have opinions on this? @leansh

Reply:
I wouldn't assume it to be unnecessary, it depends on the user, apart from that the change looks good to me